### PR TITLE
[SD-4644] (fix) Item created from template should have current date for dateline

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

File: scripts/superdesk-workspace/content/content.js [hunk 1]
```diff
@@ -147,6 +147,11 @@
         this.createItemFromTemplate = function(template) {
             var item = newItem(template.data.type || null);
             angular.extend(item, templates.pickItemData(template.data || {}), {template: template._id});
+            // set the dateline date to default utc date.
+            if (item.dateline && item.located) {
+                item.dateline = _.omit(item.dateline, 'text');
+                item.dateline.date = $filter('formatDateTimeString')();
```
Comment (petrjasek) [inline scripts/superdesk-workspace/content/content.js]: just wondering - does it set current time if used without param?

**Comment by petrjasek** [comment]
:+1:

**Comment by marwoodandrew** [comment]
:+1:

**Pull request merged**